telemetry/packages/server/src/modules/openmct/faults/data/historical/HistoricalFaultData.service.ts

```diff
@@ -67,15 +57,19 @@ export class HistoricalFaultDataService {
       const data =
         await this.influxService.query.collectRows<InfluxFaultRow>(query);
       return data.map((row) => ({
+        faultId: row['faultId'],
         timestamp: new Date(row['_time']).getTime(),
-        fault: JSON.parse(row['_value']) as OpenMctFault,
+        openMctFault: JSON.parse(row['_value']) as OpenMctFault,
+        podId: row['podId'],
+        measurementKey: row['measurementKey'],
       }));
     } catch (e: unknown) {
       this.logger.error(
         `Failed to get faults for pod ${podId}`,
```

Comment:
ESLint warning: "Invalid type "string | undefined" of template literal expression".
`Failed to get faults for pod ${podId ? podId : ''}`,